docs/Configuration.md

```diff
@@ -205,22 +205,22 @@ language from being allowed. If there are more than one `locale tag`
 all those must be removed to block that language.
 
 English is the Zonemaster default language, but it can be blocked
-from being allowed by RPC-API by not including it in the
-configuration.
+from being allowed by RPC-API by including some `locale tag` it in the
```

Comment:
I think the `it` should be removed with this update.

Reply:
Fixed.